sqlglot/expressions.py

```diff
@@ -202,6 +202,11 @@ class Unnest(Expression):
     arg_types = {'expressions': True, 'ordinality': False, 'table': False, 'columns': False}
 
 
+class Update(Expression):
+    token_type = TokenType.UPDATE
+    arg_types = {'this': True, 'set': True, 'where': False}
```

Comment:
maybe change set to be expressions since we're going to need to do CSV

Reply:
Sure, I've changed it to expressions